kuma/wiki/jinja2/wiki/translate.html

```diff
@@ -10,7 +10,6 @@
 {% block bodyclass%}translate{% endblock %}
 
 {% block extrahead %}
-  {% stylesheet 'home' %}
 {% endblock %}
```

Comment:
Do we need to keep the block around even though we are not adding any `extrahead` anymore?